api.md

```diff
@@ -779,7 +779,237 @@ Get all chunks older than 3 months and newer than 4 months:
+
+
+#### Optional Arguments [](remove_reorder_policy-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `if_exists` | (BOOLEAN)  Set to true to avoid throwing an error if the reorder_policy does not exist. A notice is issued instead. Defaults to false. |
+
+
+#### Sample Usage [](remove_reorder_policy-examples)
+
+
+```sql
+SELECT remove_reorder_policy('conditions', if_exists => true);
+```
+
+removes the existing reorder policy for the `conditions` table if it exists.
+
+---
+
+
+## alter_job_schedule() [](alter_job_schedule)
+
+Policy jobs are scheduled to run periodically via a job run in a background
+worker. You can change the schedule using `alter_job_schedule`. To alter an
+existing job, you must refer to it by `job_id`. The `job_id` which implements a
+given policy and its current schedule can be found in views in the
+`timescaledb_information` schema corresponding to different types of policies or
+in the general `timescaledb_information.policy_stats` view. This view
+additionally contains information about when each job was last run and other
+useful statistics for deciding what the new schedule should be.
+
+>:TIP: Altering the schedule will only change the frequency at which the background worker checks the policy. If you need to change the data retention interval or reorder by a different index, you'll need to remove the policy and add a new one.
+
+#### Required Arguments [](alter_job_schedule-required-arguments)
+
+|Name|Description|
+|---|---|
+| `job_id` | (INTEGER) the id of the policy job being modified | 
+
+
+#### Optional Arguments [](alter_job_schedule-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `schedule_interval` | (INTERVAL)  The interval at which the job runs |
+| `max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
+| `max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
+| `retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure | 
+| `if_exists` | (BOOLEAN)  Set to true to avoid throwing an error if the job does not exist, a notice will be issued instead. Defaults to false. | 
+
+#### Returns [](alter_job_schedule-returns)
+
+|Column|Description|
+|---|---|
+| `schedule_interval` | (INTERVAL)  The interval at which the job runs |
+| `max_runtime` | (INTERVAL) The maximum amount of time the job will be allowed to run by the background worker scheduler before it is stopped |
+| `max_retries` | (INTEGER)  The number of times the job will be retried should it fail |
+| `retry_period` | (INTERVAL) The amount of time the scheduler will wait between retries of the job on failure | 
+
+#### Sample Usage [](alter_job_schedule-examples)
+
+```sql
+SELECT alter_job_schedule(job_id, schedule_interval => INTERVAL '2 days') FROM timescaledb_information.reorder_policies WHERE hypertable = 'conditions';
```

Comment:
Should you put an actual `job_id` in the arguments of the function?

Reply:
I guess I'm confused, would the actual sql command here have, literally `job_id` in the arguments?